tests/infrastructure/vm_console_proxy/utils.py

```diff
@@ -55,7 +57,7 @@ def create_vnc_console_token(
         raise
 
 
-def get_vm_console_proxy_resource(resource_kind, namespace=None):
+def get_vm_console_proxy_resource(resource_kind: Type, namespace: Optional[str] = None) -> Type:
```

Comment:
Same comment here. No need to have Optional. For resource_kind, you can only have two options from what I see. So please specify and use `|`

Reply:
resource_kind: is list of 3 cluster and  list of 5 namespaced resources